tutorials/fmc/BUILD

```diff
@@ -0,0 +1,27 @@
+load("//bazel:defs.bzl", "cc_firmware")
+load("//bazel/tools:defs.bzl", "can_api_files")
+
+cc_firmware(
+    name = "fmc",
+    srcs = [
+        "fmc.c",
+        "fmc.h",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":can_api",
+        "//libs/adc",
+        "//libs/timer",
+    ],
```

Comment:
You might need to add `target_compatible_with`. Check other BUILD files for firmware for examples.